modules/social_features/social_node/social_node.module

```diff
@@ -21,5 +23,77 @@ function social_node_entity_type_alter(array &$entity_types) {
     foreach (['default', 'edit'] as $operation) {
       $entity_types['node']->setFormClass($operation, SocialNodeForm::class);
     }
+    $entity_types['node']->setHandlerClass('query_access', NodeQueryAccessHandler::class);
+  }
+}
+
+/**
+ * Implements hook_query_TAG_alter().
+ */
+function social_node_query_entity_query_alter(SelectInterface $query) {
```

Comment:
@rollki Could you add some more comments here?

Reply:
Sorry @bramtenhove, didn't have time yesterday to add inline comments here, but today we will have an internal meeting with the team where we will have a brainstorm and discuss some improvements (add group coverage, resolve comments issue), and I think I will try to add that improvements and at that time comment the code.